### DISCUSSION TITLE
Animations for ModelEngine 3

### DISCUSSION THREAD

**Discussion by RedxAx** [Kas-tle/java2bedrock.sh#18814] [Feature Requests]
it is possable to port the model engine legacy to bedrock, but it seems that the animation are not porting, it will be VERY helpful to implomet this! (even tho the model is kinda missed up in Bedrock).
![image](https://github.com/Kas-tle/java2bedrock.sh/assets/144875043/ef0ada61-c80a-48b7-83ba-cd1e07248430)

**Comment by HotsukokuAito** [comment]
Try to download geyseroptionalpack resource pack and put it in your packs folder in geyser
